src/main/java/org/dependencytrack/model/AnalysisLevel.java

```diff
@@ -0,0 +1,7 @@
+package org.dependencytrack.model;
+
+public enum AnalysisLevel {
```

Comment:
I think it'd be good to rename this to `VulnerabilityAnalysisLevel`, because the other `Analysis*` classes in this package are related to auditing.